controllers/external-accounts.js

```diff
@@ -14,4 +14,23 @@ const addExternalAccountData = async (req, res) => {
   }
 };
 
-module.exports = { addExternalAccountData };
+const getExternalAccountData = async (req, res) => {
+  try {
+    const externalAccountData = await externalAccountsModel.fetchExternalAccountData(req.query, req.params.token);
+    if (externalAccountData.length === 0) {
+      return res.status(404).json({ message: "No data found" });
+    }
+
+    const attributes = externalAccountData[0].attributes;
```

Comment:
please add a comment here, why array index `0` item is used instead of the whole array.